package.json

```diff
@@ -98,15 +104,18 @@
     "rollup": "1.32.0",
     "rollup-plugin-babel": "4.3.3",
     "rollup-plugin-postcss": "2.2.0",
-    "rollup-plugin-terser": "5.2.0"
+    "rollup-plugin-terser": "5.2.0",
+    "rollup-plugin-typescript2": "^0.26.0",
+    "ts-jest": "^25.2.1",
+    "tslib": "^1.11.1",
+    "typescript": "^3.8.3"
```

Comment:
Can you update all of these to not have `^`?

Reply:
Done.